rfcs/000-rfc-lightweightRFCProcess.md

```diff
@@ -0,0 +1,87 @@
+# Lightweight RFC Process
+
+**Geode-RFC-0**
+
+**To be Reviewed By:** June 20th 2019
+
+**Authors:** amurmann@apache.org
+
+**Status:** **Active** | Dropped | Superseded
+
+**Superseded by:** N/A
+
+**Related:** N/A
+
+
+## Problem
+With some of our recent efforts, we have seen consensus on design decisions take a very long time. In order to be able to keep moving forward as a successful project, we need to be able to get to consensus faster and ultimately get to execution faster. Therefore I propose a slightly more formal process for us to get to consensus.
+
+We already have gotten pretty good at sharing proposals with each other. Not much should need to change with that. It’s mainly the process around it that needs to be ironed out. This document will refer to that process as the “RFC process”.
+The main questions that the process needs to address are:
+What is enough consensus?
+How can we get to consensus in a reasonable time frame?
+
+## Solution
+The proposed solution to address the problem described above is to have an individual author or even better a group of authors submit a proposal to the community in order to gather feedback and achieve consensus. The RFC follows the same format as used by this proposal.
+
+Much inspiration for this proposal and in some cases whole segments have been drawn from Phil Calçado’s [Structured RFC Process](https://philcalcado.com/2018/11/19/a_structured_rfc_process.html). This proposal adopts Calçado’s work to work for an open source project like ours. 
```

Comment:
The last sentence here is confusing. Perhaps change it to something like `This proposal adapts Calçado’s work for use in an open source project like ours.`